internal/image.go

```diff
@@ -104,7 +103,12 @@ func FindLatestBuildImage(runURI, buildURI string) (Image, error) {
 		return Image{}, fmt.Errorf("expected the run image to be tagged but it was not")
 	}
 
-	suffix := fmt.Sprintf("-%s", tagged.Tag())
+	var suffix string
```

Comment:
Could we also move this section closer to where `suffix` is consumed? I.e. just above the `var versions []*semver.Version` line?

Reply:
on second thoughts I don't think that is any more or less clear. Happy to keep it where it is.